application/container-builder/main.go

```diff
@@ -1,42 +1,57 @@
 package main
 
-// go-git is a highly extensible git implementation in pure Go.
-// https://github.com/go-git/go-git
-
 import (
-	"fmt"
+	"context"
 
-	"github.com/go-git/go-git/v5"
+	"github.com/null-channel/eddington/application/container-builder/internal/containers/dockerfile"
+	"github.com/sirupsen/logrus"
 )
 
+// go-git is a highly extensible git implementation in pure Go.
+// https://github.com/go-git/go-git
+
 // Basic example of how to clone a repository using clone options.
 func main() {
-	url := "https://github.com/null-channel/eddington.git"
-	directory := "./tmp/eddington"
+	// url := "https://github.com/null-channel/eddington.git"
+	// directory := "./tmp/eddington"
 
-	r, err := git.PlainClone(directory, false, &git.CloneOptions{
-		URL:               url,
-		RecurseSubmodules: git.DefaultSubmoduleRecursionDepth,
-	})
+	// r, err := git.PlainClone(directory, false, &git.CloneOptions{
+	// 	URL:               url,
+	// 	RecurseSubmodules: git.DefaultSubmoduleRecursionDepth,
+	// })
 
-	if err != nil {
-		panic(err)
-	}
-	// ... retrieving the branch being pointed by HEAD
-	ref, err := r.Head()
+	// if err != nil {
+	// 	panic(err)
+	// }
+	// // ... retrieving the branch being pointed by HEAD
+	// ref, err := r.Head()
 
-	if err != nil {
-		panic(err)
-	}
+	// if err != nil {
+	// 	panic(err)
+	// }
+
+	// fmt.Println(ref)
 
-	fmt.Println(ref)
+	// // ... retrieving the commit object
+	// commit, err := r.CommitObject(ref.Hash())
 
-	// ... retrieving the commit object
-	commit, err := r.CommitObject(ref.Hash())
+	// if err != nil {
```

Comment:
can we just delete all the "bad" code that maybe I wrote? We can talk about how we get access to the code that we build the docker container with later?